app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,17 @@
+class VotesController < ApplicationController
+  def new
+    @vote = Vote.new
+  end
+
+  def create
+
```

Comment:
You should probably be creating your votes here rather than in the q/a controllers.